frontend/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
I miss from `cat`
```yaml
"trailingComma": "es5"
```

Reply:
Please see the conversation in https://github.com/src-d/company/pull/127